spec/features/person/person_remarks_spec.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2024, Schweizer Alpen-Club. This file is part of
+#  hitobito_sac_cas and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_sac_cas.
+
+require 'spec_helper'
+
+describe 'person sac remarks' do
+  let(:employee) { people(:mitglied) }
```

Comment:
es ist etwas verwirrend wenn du hier employee bezeichnest aber das mitglied fixture anziehst.
kannst du nicht gleich einfach einen Employee mit dem Fabricator erstellen?
sowas wie Fabricate(Group::...).person ... das erstellt die Rolle und gleich eine Person dazu.
gibt diverse Beispiele in bestehenden Specs

Reply:
komisch, mit `Fabricate(Group::Geschaeftsstelle::Mitarbeiter.name.to_sym, group: groups(:geschaeftsstelle)).person` geht nachher die `Group.find(person.group.id)` nicht